backup-restore/hotfixes/2.7.2/applypatch-hci-272.sh

```diff
@@ -67,7 +67,7 @@ echo "Saving original guardian-dm-operator yaml"
 oc get csv -n $BR_NS ${DM_CSV} -o yaml > guardian-dm-operator.v2.7.2-original.yaml
 oc get configmap  -n $BR_NS guardian-dm-image-config -o yaml > guardian-dm-image-config-original.yaml
 echo Updating data mover image...
-oc set data -n $BR_NS cm/guardian-dm-image-config DM_IMAGE=cp.icr.io/cp/fbr/guardian-datamover@sha256:5873062a347d02e12b74c9aa98d53d35778370ad33ce3d6115362da2c89ba71a
+oc set data -n $BR_NS cm/guardian-dm-image-config DM_IMAGE=icr.io/cpopen/guardian-datamover@sha256:1f72a26a9f9e5cb732c8447e5aad40a4574a59d13b80c45d4405b6cf3c61ffa0
```

Comment:
This does not look correct, this image is old, probably original one. probably need to be icr.io/cpopen/guardian-datamover@sha256:5873062a347d02e12b74c9aa98d53d35778370ad33ce3d6115362da2c89ba71a

Reply:
I think you are right......
```
[labuser@cube19 ~]$ skopeo inspect docker://icr.io/cpopen/guardian-datamover@sha256:5873062a347d02e12b74c9aa98d53d35778370ad33ce3d6115362da2c89ba71a | grep date
    "build-date": "2023-10-18T11:46:11",
    "com.ibm.fusion.build_date": "2024-05-01T14:46:24+00:00",
    "io.k8s.description": "The Universal Base Image Minimal is a stripped down image that uses microdnf as a package manager. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
[labuser@cube19 ~]$ skopeo inspect docker://icr.io/cpopen/guardian-datamover@sha256:1f72a26a9f9e5cb732c8447e5aad40a4574a59d13b80c45d4405b6cf3c61ffa0%7Cgrep date
    "build-date": "2023-10-18T11:46:11",
    "com.ibm.fusion.build_date": "2024-03-02T21:39:29+00:00",
    "io.k8s.description": "The Universal Base Image Minimal is a stripped down image that uses microdnf as a package manager. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
```

Reply:
I'll wait and see if this can be confirmed with Madeva and Ajesh .